### ISSUE TITLE
svg icon

### ISSUE THREAD

**Issue by ghost** [walking-octopus/vaults-ut#2]
You could use [this](https://github.com/snwh/suru-icon-theme/blob/master/src/fullcolor/apps/passwords-app.svg) icon instead of the png icon

**Comment by aparatext** [comment]
Yes, but it would have to be squared. I don't know anything about vector graphics, so I couldn't replace that curve with a square for all the layers of shadow. If you know, I'll be grateful for a PR.
